<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourierWidget</class>
 <widget class="QWidget" name="FourierWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FourierWidget</string>
  </property>
  <widget class="QPushButton" name="pbt1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>打开图片(.bmp)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入参数后确认.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ped1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="plainText">
    <string>n:每个像素rgb转hsv
小于n的视为图形，反之视为背景</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ped2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="plainText">
    <string>k:周转圆的频率,会从[-k,k]取值.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
